src/Mono.Android/Android.Graphics/Color.cs

```diff
@@ -416,7 +416,7 @@ public override void DestroyGenericArgumentState (Color value, ref JniValueMarsh
 
 		public override Expression CreateParameterToManagedExpression (JniValueMarshalerContext context, ParameterExpression sourceValue, ParameterAttributes synchronize, Type targetType)
 		{
-			var c = typeof (Color).GetConstructor (new[]{typeof (int)});
+			var c = typeof (Color).GetConstructor (new[]{typeof (int)})!;
```

Comment:
Should this be using `!` or should it instead use `?? throw new NotSupportedException ("Constructor was linked away!")`?